### PR TITLE
Patternfly upgraded to v3.10.0

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Upgrading to Patternfly and Angular-Patternfly 3.10.0 in preparation for navigation changes.

### CODE REVIEW

**Comment by dtaylor113** [comment]
:exclamation:  Please to not merge this until I test how/if this effects Blueprints.

**Comment by dtaylor113** [comment]
:thumbsup:  Ok, tested blueprint's use of bootstrap treeview, select dropdowns, filters, sorting, drag-n-drop, designer toolbox and canvas ops.  Everything looks good.

**Comment by jeff-phillips-18** [comment]
@skateman @epwinchell @himdel Please review

File: client/index.html [hunk 1]
```diff
@@ -90,12 +89,14 @@
...
   <script src="/bower_components/moment-timezone/builds/moment-timezone-with-data-2010-2020.js"></script>
   <script src="/bower_components/eonasdan-bootstrap-datetimepicker/build/js/bootstrap-datetimepicker.min.js"></script>
+  <script src="/bower_components/patternfly-bootstrap-treeview/dist/bootstrap-treeview.min.js"></script>
   <script src="/bower_components/patternfly/dist/js/patternfly.js"></script>
   <script src="/bower_components/angular-patternfly/dist/angular-patternfly.js"></script>
   <script src="/bower_components/angular-drag-and-drop-lists/angular-drag-and-drop-lists.js"></script>
   <script src="/bower_components/angular-resource/angular-resource.js"></script>
   <script src="/bower_components/angular-svg-base/src/svgDirs.js"></script>
   <script src="/bower_components/angular-ui-router/release/angular-ui-router.js"></script>
+  <script src="/bower_components/bootstrap-treeview/dist/bootstrap-treeview.min.js"></script>
```
Comment (himdel) [inline client/index.html]: Should we be depending on both bootstrap-treeview **and** patternfly-bootstrap-treeview? Seems like one is a fork of the other, right?

File: bower.json [hunk 2]
```diff
@@ -53,5 +52,9 @@
     "spice-html5-bower": {
       "main": []
     }
+  },
+  "resolutions": {
```
Comment (himdel) [inline bower.json]: Since we're running bower with `-F`, we *probably* don't need to add the resolutions section, right? Or do we?

**Comment by himdel** [comment]
Not seeing any problems in the UI, so .. I'd merge, except I'm not sure about that `bootstrap-treeview` vs `patternfly-bootstrap-treeview` problem..

**Comment by jeff-phillips-18** [comment]
@himdel I've updated bower.json

**Comment by himdel** [comment]
Thanks @jeff-phillips-18 , merging when green.. :)

**Pull request merged**